tests/x11regressions/tracker/tracker_by_command.pm

```diff
@@ -17,12 +17,9 @@ use testapi;
 sub run() {
     my $self = shift;
     x11_start_program("xterm");
-    sleep 2;
-    wait_idle;
     script_run "tracker-search newfile";
-    sleep 5;
     assert_screen 'tracker-cmdsearch-newfile';
-    script_run "exit";
+    send_key 'ctrl-d';
```

Comment:
I guess the same applies here as in before for "ctrl-l" vs. "clear". The control character will only act upon a shell prompt which could not be ensured in before so "ctrl-d" is unsafe, I guess.